### PR TITLE
cmake: Whitespace and commentary fixes

### PR DESCRIPTION
**Pull request by mped-oticon**
Cosmetics, no functional change expected.

- Fixed leading space alignment.
- Replaced tabs with spaces.
- Emulation error message output is now aligned.

### CODE REVIEW

File: cmake/backports/FindPythonInterp.cmake [hunk 1]
```diff
@@ -131,12 +131,12 @@ if(NOT PYTHON_EXECUTABLE)
         list(APPEND _Python_NAMES python)
       endif()
 
-	  if(WIN32)
+    if(WIN32)
```
Comment (SebastianBoe) [inline cmake/backports/FindPythonInterp.cmake]: I would expect the 'endif' on line 132 to be at the same alignment as this 'if' statement.

Is there some more whitespace issues that need fixing here?

The same applies to line 139 & 141.

File: cmake/dts.cmake [hunk 2]
```diff
@@ -36,23 +36,23 @@ foreach(board_root ${BOARD_ROOT})
   # we construct a list of shield names by extracting file name and
   # removing the extension.
   foreach(shield_path ${shields_refs_list})
-	get_filename_component(shield ${shield_path} NAME_WE)
+  get_filename_component(shield ${shield_path} NAME_WE)
```
Comment (SebastianBoe) [inline cmake/dts.cmake]: 'get_filename_component' should be indented relative to 'foreach'

File: cmake/dts.cmake [hunk 3]
```diff
@@ -73,13 +73,13 @@ if(CONFIG_HAS_DTS)
...
-	  message(STATUS "Overlaying ${dts_file}")
-	endif()
+  if(i EQUAL 0)
+    message(STATUS "Loading ${dts_file} as base")
+  else()
+    message(STATUS "Overlaying ${dts_file}")
+  endif()
 
-	math(EXPR i "${i}+1")
+  math(EXPR i "${i}+1")
```
Comment (SebastianBoe) [inline cmake/dts.cmake]: 'math' should be indented relative to 'endforeach'.

**Review by SebastianBoe — CHANGES_REQUESTED** [review]
This is great overall.

Minor tweaks needed.

Which tool do you use to adjust the whitespace?

**Comment by mped-oticon** [comment]
> Which tool do you use to adjust the whitespace?

I use grep + sublime text :) Fixing it in sublime is easy enough with multiple cursors.

I have also tried https://github.com/cheshirekow/cmake_format but I find it changes too much.

**Comment by mped-oticon** [comment]
Review comments fixed.

**Review by SebastianBoe — APPROVED** [review]

**Review by carlescufi — APPROVED** [review]

**Pull request merged**